edalize/veriblelint.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import re
+import os
+import subprocess
+
+from edalize.edatool import Edatool
+
+logger = logging.getLogger(__name__)
+
+class Veriblelint(Edatool):
+
+    argtypes = ['vlogdefine', 'vlogparam']
+
+    @classmethod
+    def get_doc(cls, api_ver):
+        if api_ver == 0:
+            return {'description' : "Verible lint backend (verilog_lint)",
+                    'members': [
+                         {'name': 'ruleset',
+                          'type': 'String',
+                          'desc': 'Ruleset: [default|all|none]'},
+                    ],
+                    'lists': [
+                         {'name' : 'verible_lint_args',
+                         'type' : 'String',
+                         'desc' : 'Extra command line arguments passed to the Verible tool'},
+                         {'name': 'rules',
+                          'type': 'String',
+                          'desc': 'What rules to use. Prefix a rule name with "-" to disable it.'},
+                    ]}
+
+
+    def build_main(self):
+        pass
+
+    def _get_tool_args(self):
+        args = [ '-lint_fatal', '-parse_fatal' ]
```

Comment:
Yes, these are the options you want.  (These weren't default because the infrastructure this plugged into wanted exit-status 0.)